### ISSUE TITLE
Enhancement: Remote Device Management (RDM)

### ISSUE THREAD

**Issue by 303Bryan** [Breina/ha-artnet-led#24]
It would be cool to see support for [DMX RDM](https://en.wikipedia.org/wiki/RDM_(lighting)). Some of the fancier lights support this method for discovery and device management. This is likely a bottom-of-the-list item ahead of your other work to streamline the integration, but I thought I would through it out for future consideration.

**Comment by Breina** [comment]
Agreed!

This will indeed be low-prio, first I want to get everything else working. I don't have any RDM devices personally, can I enrol you as tester for when this gets implemented?

**Comment by 303Bryan** [comment]
Yes! Happy to help with testing. I've got 8 RDM-capable lights connected to my setup.
